.eslintrc.yaml

```diff
@@ -16,32 +18,15 @@ extends:
   - plugin:@next/next/recommended
   - prettier
 
-plugins:
-  - prettier
-
-globals:
-  "cy": true
-
 rules:
-  prettier/prettier: warn
-
-  react/react-in-jsx-scope: 0
-  react/jsx-filename-extension: 0
-  react/no-unescaped-entities: 0
-  react/jsx-props-no-spreading: 0
-  react/require-default-props: 0
-  react/no-children-prop: 0
+  react/react-in-jsx-scope: off
+  react/jsx-filename-extension: off
+
   react-hooks/exhaustive-deps: off
```

Comment:
This is a react warning so it should be detected?

Reply:
If enabled we can't use an `useEffect` hook with an empty array (or we should move the variable declarations inside it)

We could talk about it, looks better like that imo but it's all about preferences 👍

Reply:
Didn't know that trick 😉 Otherwise I think it still **musts** detect useEffects dependencies outside of the array